pkg/liveshare/session.go

```diff
@@ -27,6 +29,10 @@ type Session struct {
 	logger          logger
 }
 
+type StartSSHServerOptions struct {
+	userPublicKeyFile string
+}
```

Comment:
This is a bit weird to export a public type but has no public values. I assume it is because we expect callers to use the option setters defined in this package, but if we made it public, callers could define their own config setters and make it a bit more extensible?